lib/driver.js

```diff
@@ -473,10 +473,7 @@ class AndroidDriver extends BaseDriver {
   }
 
   getPrefsBuilder () {
```

Comment:
@jlipps I tried, but couldnt find a way to mock `new SharedPrefsBuilder`. I was going to ask you guys today